.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: node_js
 node_js:
-  - "6.1"
+  - "6.9"
```

Comment:
usually I always keep the latest of the major release so in this case 6 and 7, I changed the version to run Travis on since those 2 were the latest for 6 and 7 when I was testing